microsoft/devhome
# Filling TextBox on user input only

## Summary of the pull request
An AutoSuggestBox will provide an enum that tells the event who invoked the event.  DevHome was not using this enum.  This caused the following issue

1. Using the up/down arrow keys would fill in the text box _and_ initiate filtering leading to all the suggestions being removed because the filter looked for all suggestions that was put into the text box (The string the user navigated to).

meaning keyboard navigation was impossible.

The fix was to listen and not update the enum if the user navigated the suggestions.

## References and relevant issues

## Detailed description of the pull request / Additional comments

https://task.ms/51630446

## Validation steps performed

Manually tested.  Set a breakpoint in the azure extension to verify values were passed into the extension.

## PR checklist
